python/python/tests/test_async_client.py

```diff
@@ -5912,13 +6054,13 @@ async def test_pubsub_basic_standalone(self, request):
         pattern = None
         for _ in range(2):
             pubsub_msg = await listening_client.get_pubsub_message()
-            assert pubsub_msg.channel == CHANNEL_NAME
-            assert pubsub_msg.message == MESSAGE
+            assert pubsub_msg.channel == CHANNEL_NAME.encode()
```

Comment:
channel must be left as str, why do we need it?

Reply:
The next PR will change some commands that return strings